tools/orca/src/generate.rs

```diff
@@ -146,15 +147,23 @@ pub async fn populate(_client: &KanidmOrcaClient, profile: Profile) -> Result<St
     // them a baseline of required accounts with some variation. This
     // way in each test it's guaranteed that *at least* one person
     // to each role always will exist and be operational.
+    let groups_person_count_map: HashMap<&String, u64> = profile
```

Comment:
I don't understand what this is doing?

Reply:
This is derived from `person_count_by_group`. The idea is that for each group we check if its name is also a key inside the hashmap. If it is we set the number of members of that group to the value stored in the hashmap under its name. When a group name is not a key, then we randomly choose the number of members it should have, like we did before.